yoti-sdk-impl/src/test/java/com/yoti/api/client/shareurl/policy/SimpleWantedAttributeBuilderTest.java

```diff
@@ -55,4 +55,9 @@ public void buildsAnAttributeWithSourceConstraint() {
         assertEquals(result.getConstraints(), constraintListMock);
     }
 
+    @Test(expected = IllegalArgumentException.class)
+    public void buildingAnAttributeWithoutNameFails() {
+        WantedAttributeBuilder.newInstance().build();
+    }
```

Comment:
Can we change this test to check that the `IllegalArgumentException` message contains `name`?  I think `builder_shouldApplyMaxLimitToDay()` in `DateValueTest.java` is a good example of how to structure it